### PR TITLE
Don't check for updates in the guest session

### PR DESCRIPTION
**Pull request by meisenzahl**
Closes https://github.com/elementary/appcenter/issues/2088

### CODE REVIEW

**Comment by danirabbit** [comment]
I think we probably want to also remove or disable the "Check for updates" button, and possibly avoid showing the update list entirely in the installed view

**Comment by zeebok** [comment]
If you do that, you might want to just remove the settings menu since the other option pertains to installing updates

**Comment by meisenzahl** [comment]
Thank you very much for your feedback. @danirabbit could you please explain again in more detail what you mean by:

> possibly avoid showing the update list entirely in the installed view

File: src/MainWindow.vala [hunk 1]
```diff
@@ -212,8 +212,12 @@ public class AppCenter.MainWindow : Hdy.ApplicationWindow {
         };
         headerbar.set_custom_title (search_clamp);
         headerbar.pack_start (return_button);
-        headerbar.pack_end (menu_button);
-        headerbar.pack_end (view_mode_revealer);
+        if (!Utils.is_running_in_guest_session ()) {
+            headerbar.pack_end (menu_button);
+            headerbar.pack_end (view_mode_revealer);
+        } else {
+            headerbar.pack_end (new Gtk.Label (""));
```
Comment (meisenzahl) [inline src/MainWindow.vala:219]: If I do not call `headerbar.pack_end`, the headerbar is broken. Is there a better way than adding an empty widget?

**Review by danirabbit — APPROVED** [review]

**Pull request merged**